ipv4_test.go

```diff
@@ -5,7 +5,7 @@ import (
 	"testing"
 	"time"
 
-	"github.com/songgao/water/waterutil"
+	"./waterutil"
```

Comment:
Relative import paths don't work well in Go. It'd be a bigger problem if this weren't a `_test.go` file since it'd break `go install`. But this is still an issue. For example, you can't do stuff like `go test github.com/songgao/water` anymore since it won't build.

Reply:
This is indeed a problem, because having the full GitHub URL means that it isn't possible to fork `water` (like I did) and then refer to it from a project (like I did) because eventually `songgao/water` gets pulled back in and then fails to build due to attempted redefinitions. I do not know quite how to fix it apart from removing that absolute link.

Reply:
I don't think it's common to use a relative import path in Go. If you need to use your own fork, a widely used solution is to use a vendoring tool (which you normally want to anyway, to pin on git commit hashes that you know are compatible), where you can specify the `origin` path to be your own fork, while still use the package name `github.com/songgao/water`. For example for `govendor`, `origin` is the path where code is from, and `path` is the package path.